### PR TITLE
chore: updating translations in French and Swahili 

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Compared to the English file, I have added and removed translation keys to match the English one. 
The files have the same number of translations keys:  1449 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:adding-missing-translations/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:adding-missing-translations/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:adding-missing-translations/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@m5r @Benmuiruri can you please review? I used Google translate

File: api/resources/translations/messages-sw.properties [hunk 1]
```diff
@@ -1342,6 +1328,7 @@ task.days.left =
 task.immunization_missing_visit.title = Ziara ya chanjo inakosekana
 task.list.complete = Hakuna jambo ingine
 task.overdue = Imepita siku ya kuwasilishwa
+task.overdue.days = {DAYS, plural, =0{Inadaiwa leo} =1{Ilidaiwa jana} other{Inadaiwa siku \# zilizopita}}
```
Comment (Benmuiruri) [inline api/resources/translations/messages-sw.properties:1331]:
```suggestion
task.overdue.days = {DAYS, plural, =0{Inatarajiwa leo} =1{Ilitarajiwa jana} other{Ilitarajiwa siku \# zilizopita}}
```

**Review by Benmuiruri — APPROVED** [review]
1 small change

File: api/resources/translations/messages-fr.properties [hunk 2]
```diff
@@ -686,6 +687,8 @@ export.feedback.description = Téléchargez un journal des erreurs détectées e
 export.messages.description = Téléchargez tous les messages qui ont déjà été envoyés ou reçus au format CSV.
 export.people.description = Téléchargez tous les contacts enregistrés dans le système au format JSON.
 export.reports.description = Téléchargez un résumé de tous les rapports qui ont déjà été soumis au format CSV.
+export.tabs.user_devices = Appareils utilisateur
+export.user-devices.description = Téléchargez un résumé des versions logicielles liées à CHT (navigateur, instance CHT et système d'exploitation) exécutées sur les appareils de l'utilisateur. Le résumé est exporté au format JSON.
```
Comment (m5r) [inline api/resources/translations/messages-fr.properties:691]:
```suggestion
export.user-devices.description = Téléchargez un résumé des versions des logiciels liés au CHT (navigateur, instance CHT et système d'exploitation) exécutés sur les appareils de l'utilisateur. Le résumé est exporté au format JSON.
```

**Review by m5r — APPROVED** [review]
Thanks Jenni! Just one wording change, approving so that you can merge when you've made the change

**Pull request merged**